package.json

```diff
@@ -5,9 +5,9 @@
   "license": "MIT",
   "dependencies": {
     "@daostack/client-experimental": "1.0.0-rc.1",
-    "@daostack/migration-experimental": "0.1.1-rc.12-v0",
+    "@daostack/migration-experimental": "0.1.1-rc.13-v1",
```

Comment:
this can be totally excluded . as all addresses ,include daoFactory can be fetched from subgraph.
no need to import migration ,